src/vec.ts

```diff
@@ -153,44 +149,31 @@ export namespace vec {
   }
 
   /**
-   * Inverts a 3x3 matrix, returning null if the determinant is zero (indicating a degenerate
-   * transformation)
+   * Inverts a 3x3 matrix, returning null if the determinant is zero
+   * (indicating a degenerate transformation)
    */
-  export function matrixInvert(matrix: Matrix): Matrix | null {
-    const a = matrix
-    const out: Matrix = new Array(9) as Matrix
+  export function matrixInvert(a: Matrix): Matrix | null {
```

Comment:
same as above, I did this by find-replacing the `0` and `1` entries and simplifying.